pkg/memtier/pidwatcher_cgroups.go

```diff
@@ -198,10 +198,7 @@ func (w *PidWatcherCgroups) loop(singleshot bool) {
 		}
 
 		// Wait for next tick.
-		select {
-		case <-ticker.C:
-			continue
-		}
+		_ = ticker.C
```

Comment:
This will not wait for the next tick, but it really should...

Maybe we should add some e2e test a check that `PidWatcherCgroups.loop` heartbeat rate looks healthy....